<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>934</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>We Do Not Allow Any Horniness In Here!</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Catch Cheems If You Can</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>753</width>
          <height>118</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>How fast will Cheems go </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtTime"/>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>(ms)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="butClear">
             <property name="text">
              <string>Let me save Cheems again</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Horny times</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Bonk! times</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QLineEdit" name="txtHorny"/>
               </item>
               <item>
                <widget class="QLineEdit" name="txtBonk"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QPushButton" name="butStart">
               <property name="text">
                <string>Start being horny</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="butStop">
               <property name="text">
                <string>Stop that shit</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>240</y>
          <width>180</width>
          <height>180</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <activeoff>:/imgs/offHorny.png</activeoff>
          <activeon>:/imgs/onHorny.jpg</activeon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>640</width>
          <height>640</height>
         </size>
        </property>
        <property name="shortcut">
         <string extracomment="i said no horny"/>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="labOuch">
        <property name="geometry">
         <rect>
          <x>710</x>
          <y>160</y>
          <width>161</width>
          <height>101</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>40</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text">
         <string>Ouch!</string>
        </property>
       </widget>
       <zorder>pushButton</zorder>
       <zorder>layoutWidget</zorder>
       <zorder>labOuch</zorder>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Cheems In a Hole</string>
       </attribute>
       <widget class="QLabel" name="labOuch_2">
        <property name="geometry">
         <rect>
          <x>890</x>
          <y>160</y>
          <width>161</width>
          <height>101</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>40</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text">
         <string>Ouch!</string>
        </property>
       </widget>
       <widget class="QLabel" name="txtCheems">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>240</y>
          <width>180</width>
          <height>180</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resources.qrc">:/imgs/offHorny.png</pixmap>
        </property>
       </widget>
       <widget class="QLabel" name="txtCheems_2">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>240</y>
          <width>180</width>
          <height>180</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resources.qrc">:/imgs/offHorny.png</pixmap>
        </property>
       </widget>
       <widget class="QLabel" name="txtCheems_3">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>240</y>
          <width>180</width>
          <height>180</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resources.qrc">:/imgs/offHorny.png</pixmap>
        </property>
       </widget>
       <widget class="QLabel" name="txtCheems_4">
        <property name="geometry">
         <rect>
          <x>570</x>
          <y>430</y>
          <width>180</width>
          <height>180</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resources.qrc">:/imgs/offHorny.png</pixmap>
        </property>
       </widget>
       <widget class="QLabel" name="txtCheems_5">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>430</y>
          <width>180</width>
          <height>180</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resources.qrc">:/imgs/offHorny.png</pixmap>
        </property>
       </widget>
       <widget class="QPushButton" name="butCheems">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>240</y>
          <width>180</width>
          <height>180</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>180</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="butCheems_2">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>240</y>
          <width>180</width>
          <height>180</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>180</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="butCheems_3">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>240</y>
          <width>180</width>
          <height>180</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>180</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>831</width>
          <height>118</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Cheems will show up every </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtTime_2"/>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>(ms)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="butClear_2">
             <property name="text">
              <string>Give me another chance to protect Cheems</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>Horny counts</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>Ouch! counts</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <widget class="QLineEdit" name="txtHorny_2"/>
               </item>
               <item>
                <widget class="QLineEdit" name="txtBonk_2"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <widget class="QPushButton" name="butStart_2">
               <property name="text">
                <string>OK let's do this</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="butStop_2">
               <property name="text">
                <string>Stop that shit</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>230</y>
          <width>193</width>
          <height>193</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resources.qrc">:/imgs/ground.jpg</pixmap>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>230</y>
          <width>193</width>
          <height>193</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resources.qrc">:/imgs/ground.jpg</pixmap>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>660</x>
          <y>230</y>
          <width>193</width>
          <height>193</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resources.qrc">:/imgs/ground.jpg</pixmap>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>420</y>
          <width>193</width>
          <height>193</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resources.qrc">:/imgs/ground.jpg</pixmap>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>570</x>
          <y>420</y>
          <width>193</width>
          <height>193</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resources.qrc">:/imgs/ground.jpg</pixmap>
        </property>
       </widget>
       <widget class="QPushButton" name="butCheems_7">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>430</y>
          <width>180</width>
          <height>180</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>180</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="butCheems_8">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>430</y>
          <width>180</width>
          <height>180</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>180</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <zorder>label_13</zorder>
       <zorder>label_12</zorder>
       <zorder>label_11</zorder>
       <zorder>label_10</zorder>
       <zorder>label_9</zorder>
       <zorder>txtCheems</zorder>
       <zorder>txtCheems_2</zorder>
       <zorder>txtCheems_3</zorder>
       <zorder>txtCheems_4</zorder>
       <zorder>txtCheems_5</zorder>
       <zorder>butCheems</zorder>
       <zorder>butCheems_2</zorder>
       <zorder>butCheems_3</zorder>
       <zorder>layoutWidget_2</zorder>
       <zorder>labOuch_2</zorder>
       <zorder>butCheems_8</zorder>
       <zorder>butCheems_7</zorder>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1500</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
